File: java/src/MainBatallaCarnotronZapato.java

import model.batalla.Batalla;
import model.chimpos.*;
import model.logger.Logger;
import model.logger.Nivel;
import model.referi.Referi;
import model.tipo.TipoChinpokomon;


public class MainBatallaCarnotronZapato {

    public static void main(String[] args) {
        Logger.getInstance(Nivel.INFO);
        Carnotron carnotron = new Carnotron("Carnotron", 6, TipoChinpokomon.ROBOT);
        Zapato zapato = new Zapato("Zapato", 5, TipoChinpokomon.ANIMAL);
        Batalla batalla = new Batalla(carnotron, zapato);
        Referi referi = new Referi(batalla);
        referi.gestionarBatalla();
    }
}

File: python/MainBatallaCarnotronZapato.py
from model.logger.logger import Logger
from model.logger.nivel import Nivel
from model.chipos.Carnotron import Carnotron
from model.chipos.Zapato import Zapato
from model.chipos.enums import TiposDeChipos
from model.batalla.batalla import Batalla
from model.referi.referi import Referi

Logger.instance(Nivel.INFO)
carnotron = Carnotron("Carnotron", 6, TiposDeChipos.ROBOT)
zapato = Zapato("Zapato", 5, TiposDeChipos.ANIMAL)
batalla = Batalla(carnotron, zapato)
referi = Referi(batalla)
referi.gestionar_batalla()
